client/extensions/woocommerce/woocommerce-services/views/service-settings/settings-form/settings-item.js

```diff
@@ -99,6 +100,35 @@ const SettingsItem = ( {
 				/>
 			);
 
+		case 'shipping_classes':
+			const items = [
+				{ id: 'method-a', label: 'Method A' },
+				{ id: 'method-b', label: 'Method B' },
+				{ id: 'method-c', label: 'Method C' },
+			];
+
+			return (
+				<TokenField
+					id={ id }
+					title={ fieldSchema.title }
+					description={ fieldSchema.description }
+					value={ fieldValue }
+					suggestions={ items.map( item => item.id ) }
+					updateValue={ updateValue }
+					saveTransform={ token => {
+						token = token.trim().toUpperCase();
+						const transformed = items.find( item => item.id.toUpperCase() === token );
```

Comment:
This seems to be a commit that I somehow did not push, but I had already changed it to `.name` instead of `.id`. The reason was that I developed the input before the data layer and during that time, I was using strings as ids.